packages/jspsych/src/modules/plugin-api/MediaAPI.ts

```diff
@@ -1,7 +1,15 @@
+import { ParameterType } from "../../modules/plugins";
 import { flatten, unique } from "../utils";
 
+const preloadParameterTypes = <const>[
+  ParameterType.AUDIO,
+  ParameterType.IMAGE,
+  ParameterType.VIDEO,
+];
+type PreloadType = typeof preloadParameterTypes[number];
+
 export class MediaAPI {
-  constructor(private useWebaudio: boolean, private webaudioContext: AudioContext) {}
+  constructor(private useWebaudio: boolean = false, private webaudioContext?: AudioContext) {}
```

Comment:
Just double checking that the default for `useWebaudio` should be `false`. My memory was that we defaulted to `true` in previous releases, unless the page was loaded using the `file://` protocol.

Reply:
Very right. What we write here is just internally when we instantiate `MediaAPI` ourselves during testing; JsPsych always provides the useWebaudio parameter when it instantiates `MediaAPI`. Just didn't want to pass an audio context during testing... I agree that this is confusing. Should I remove it again and pass it in the tests?

Reply:
Yeah, I think I would rather make the test files a little more complicated than have a confusing default value here.